test/issue-28.test.js

```diff
@@ -499,3 +499,79 @@ test('Nested wildcards with parametric and static - 6', t => {
     null
   )
 })
+
+test('Nested wildcards with parametric and static - 7', t => {
+  t.plan(1)
+  const findMyWay = FindMyWay({
+    defaultRoute: (req, res) => {
+      t.fail('we should not be here, the url is: ' + req.url)
+    }
+  })
+
+  findMyWay.on('GET', '*', (req, res, params) => {
+    t.fail('we should not be here, the url is: ' + req.url)
+  })
+
+  findMyWay.on('GET', '/foo1/*', (req, res, params) => {
+    t.fail('we should not be here, the url is: ' + req.url)
+  })
+
+  findMyWay.on('GET', '/foo2/*', (req, res, params) => {
+    t.fail('we should not be here, the url is: ' + req.url)
+  })
+
+  findMyWay.on('GET', '/foo3/:param', (req, res, params) => {
+    t.is(req.url, '/foo3/hello')
+  })
+
+  findMyWay.on('GET', '/foo3/*', (req, res, params) => {
+    t.fail('we should not be here, the url is: ' + req.url)
+  })
+
+  findMyWay.on('GET', '/foo4/param/hello/test/long/route', (req, res, params) => {
```

Comment:
I find it hard reading a route like this, cause I've to figure out from the context of the test whether `param` was really thought as a static part of the route, or maybe it was meant to be a parameter, and there's a typo.
So I suggest to not use the word `param` in a long static route... we can use whatever else.

This occurs also in a couple of other tests in this file.